packages/react-ui-components/src/SelectBox/style.css

```diff
@@ -91,27 +91,30 @@
     line-height: var(--goldenUnit);
 }
 
-.selectBox__twoIconIndention {
-    .selectBox__loadingIcon::after {
-        display: block;
-        content: '';
-        position: absolute;
-        width: 1px;
-        height: 24px;
-        top: 0;
-        right: 0;
-        background-color: #fff;
-        opacity: .15;
-        margin-top: var(--halfSpacing);
-    }
+.selectBox__deleteIcon {
+    composes: selectBox__loadingIcon;
 }
 
 .selectBox__twoIconIndention {
     padding: 0 calc(var(--spacing) + var(--goldenUnit) + var(--goldenUnit)) 0 var(--spacing);
 }
 
-.selectBox__deleteIcon {
-    composes: selectBox__loadingIcon;
+.selectBox__twoIconIndention .selectBox__iconSeparator {
+    display: block;
+    content: '';
+    position: absolute;
+    width: 1px;
+    height: 24px;
+    top: 0;
+    right: 41px;
+    background-color: #fff;
+    opacity: .15;
+    margin-top: var(--halfSpacing);
+}
+
+.selectBox__twoIconIndention .selectBox__deleteIcon,
```

Comment:
You are trying to cascade, but that won't work with CSS modules (on purpose). Try to understand how CSS modules work, it's quite frustrating to work with them without understanding the principle.
All cascade-like behavior should be done on React level (e.g. via props).

Reply:
Ah, and it's not postcss, it's css modules that have a problem.

Reply:
Or am I wrong and it does work?